### PR TITLE
S2t3 option methods

### PR DESCRIPTION
**Pull request by hannahgolightly**

### CODE REVIEW

File: db/academyPortal_2020-05-19.sql [hunk 1]
```diff
@@ -5,9 +5,9 @@
 # http://www.sequelpro.com/
 # https://github.com/sequelpro/sequelpro
 #
-# Host: 127.0.0.1 (MySQL 5.7.29)
+# Host: 127.0.01 (MySQL 5.7.29)
```
Comment (HiredMussel) [inline db/academyPortal_2020-05-19.sql:8]: This was right the first time. The IP Address for localhost is 127.0.0.1

Comment (HiredMussel) [reply db/academyPortal_2020-05-19.sql:8]: Not sure what's happened here to be honest. Try dumping the db again?

File: src/Models/StageModel.php [hunk 2]
```diff
@@ -133,6 +133,30 @@ public function updateAllStages(array $stages) : bool
         }
     }
 
+    public function createOption(string $option, int $stageId) : bool
```
Comment (HiredMussel) [inline src/Models/StageModel.php:136]: Might be worth adding documentation to these functions

File: src/Models/StageModel.php [hunk 3]
```diff
@@ -133,6 +133,30 @@ public function updateAllStages(array $stages) : bool
...
+    {
+        $query = $this->db->prepare("INSERT INTO `options` (`option`, `stageId`) VALUES (:optionText, :stageId);");
+        $query->bindParam(':optionText', $option);
+        $query->bindParam(':stageId', $stageId);
+        return $query->execute();
+    }
+
+    public function updateOption(string $option, int $optionId) : bool
```
Comment (HiredMussel) [inline src/Models/StageModel.php:144]: Docblock

File: src/Models/StageModel.php [hunk 4]
```diff
@@ -133,6 +133,30 @@ public function updateAllStages(array $stages) : bool
...
+    {
+        $query = $this->db->prepare("UPDATE `options` SET `option` = :optionText WHERE `id` = :id");
+        $query->bindParam(':id', $optionId);
+        $query->bindParam(':optionText', $option);
+        return $query->execute();
+    }
+
+    public function deleteOption(int $optionId) : bool
```
Comment (HiredMussel) [inline src/Models/StageModel.php:152]: Docblock

File: src/Models/StageModel.php [hunk 5]
```diff
@@ -133,6 +133,30 @@ public function updateAllStages(array $stages) : bool
...
+    {
+        $query = $this->db->prepare("UPDATE `options` SET `deleted` = '1' WHERE `id` = :optionId");
+        $query->bindParam(':optionId', $optionId);
+        return $query->execute();
+    }
+
+
```
Comment (HiredMussel) [inline src/Models/StageModel.php:159]: Can't put this anywhere, but just for your information: this commit was failing the CircleCI tests for a bit of a silly reason, which is that CircleCI expects every file to end with a "new line" character. Keep that in mind going forward.

File: db/academyPortal_2020-05-19.sql (hunk 1 above)
Comment (hannahgolightly) [reply db/academyPortal_2020-05-19.sql]: fixed this, re exported db

File: src/Models/StageModel.php (hunk 2 above)
Comment (hannahgolightly) [reply src/Models/StageModel.php:142]: done

File: src/Models/StageModel.php (hunk 3 above)
Comment (hannahgolightly) [reply src/Models/StageModel.php:156]: done

File: src/Models/StageModel.php (hunk 4 above)
Comment (hannahgolightly) [reply src/Models/StageModel.php:169]: done

File: src/Models/StageModel.php (hunk 5 above)
Comment (hannahgolightly) [reply src/Models/StageModel.php:176]: ok fixed thanks

**Pull request merged**